indexer/indexer.toml

```diff
@@ -3,6 +3,14 @@
 [chain]
 # OP Goerli
 preset = 420
+
+[chain.l1-contracts]
+optimism-portal = ""
+l2-output-oracle = ""
+l1-cross-domain-messenger = ""
+l1-standard-bridge = ""
+l1-erc721-bridge = ""
+
```

Comment:
```suggestion
```
Will document this better but a Preset essentially means you don't provide the l1-contracts because they are already provided by the preset. Presets exist for chains like OP, base, base-goerli, and zora. The l1-contracts is simply an escape hatch for when a chain doesn't have a preset

Reply:
That makes sense, just updated the config accordingly!